14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,87 @@
+
+The API returns responses in a JSON format, which can be presented in different channels using your preferred technology stack. This feature maintains the user-friendly editing experience of Umbraco, while also ensuring an efficient delivery of content in a headless manner. Additionally, with its different extension points, the API can be customized to meet a wide range of requirements.
+
+The Management API can also be used for Custom apps or Workflows with OpenID Connect.
+
+{% hint style="info" %}
+The Management API is a replacement for the backoffice controllers that lacked RESTful capabilities.
+{% endhint %}
+
+## Swagger Documentation
+
+Umbraco ships with Swagger to document the Management API. Swagger and the Swagger UI are based on [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/) and is available at `{yourdomain}/umbraco/swagger`. For security reasons, both are disabled in non-production environments.
+Read more about Swagger in the [API versioning and OpenAPI](../api-versioning-and-openapi.md) article.
+
+The Swagger documentation allows you to select a definition and go to either Umbraco Management API or Content Delivery API. You can also create custom documentation in Swagger by referring to the [Custom Swagger API with Authorization](../custom-swagger-api.md) article.
+
+![Umbraco Management API documentation in Swagger](../images/management-api-swagger.png)
+
+In the Swagger Umbraco Management API, you can find a collection of available endpoints in this version of Umbraco. It also includes all the legacy APIs that are available for backward compatibility.
+
+## Authorization
+
+The Management API endpoints are protected by the backoffice authorization policy and need an authentication token to interact with them.
+
+To set the authorization, click on the **Authorize** button:
+
+![Umbraco Management API Authorize Button](../images/management-api-swagger-authorize-button.png)
+
+Then a popup will appear with some setup information and login form for authorization:
+
+![Umbraco Management API Authorize Login](../images/management-api-swagger-authorize-instructions.png)
+
+The available integration for the authorization is done via a backoffice user with the integration of `OAuth2, authorizationCode with PKCE`. If you are working on a production environment, you will need to setup an OAuth2 integration with the following Umbraco Authentication specifications:
+
+**Authorization URL**
+
+```http
+{yourdomain}/umbraco/management/api/v1/security/back-office/authorize
+```
+
+**Token URL**
+
+```http
+{yourdomain}/umbraco/management/api/v1/security/back-office/token
+```
+
+**Flow**
+
+```http
+authorizationCode with PKCE
+```
+
+**client_id**
+
+```http
+umbraco-swagger
+```
```

Comment:
> If you are working on a production environment, you will need to setup an OAuth2 integration with the following Umbraco Authentication specifications:

I don't really understand this. Are we trying to tell people that they can use Postman or other tools to connect to the Management API in production here? The mentioning of "production" is super confusing here.

Also, you likely won't "setup an OAuth2 integration" - if anything you'll "connect via OAuth2".